examples/dlccli/scripts/start_bitcoind.sh

```diff
@@ -43,11 +44,24 @@ function getblockcount() {
 height=$(getblockcount)
 echo "Block Height: ${height}"
 
-echo "Creating wallets"
-$bitcoincli "${opts[@]}" createwallet "alice" "false" "false" "alice"
-$bitcoincli "${opts[@]}" createwallet "bob" "false" "false" "bob"
```

Comment:
I can see there is an issue with v23, but personally I could solve it by just adding `"false" "false"` on this two lines. Could you try and see if that works? That would be a smaller diff, unless you have another reason for removing the passphrase?

Reply:
with "false" "false" was already in this file before my PR. on my machine, with v23, if you run start-bitcoin.sh for a second time, the createwallet functions fail. But then when you run the application, they aren't loaded so the DLC functions fail. So we need an open.

And then I was already going down the path of not having the passphrases so I just went that way, for a test env it seemed fine. But if you prefer I can put them back in, just need to include it in a few places, and still need to solve the issue of loadwallet if it's already created.

Reply:
What I meant was to add additionals `"false" "false"` after `"alice"` and `"bob"`, but actually I guess using named parameters as you did might actually be cleaner, and the loadwallet part is also nice. For the passphrase, the reason I used it is because I think it's a good practice and so felt like it was a good idea to use it for the tutorial, I don't have that strong of an opinion but I'd be more inclined to keep it.

Reply:
Cool, I'm pretty new to BTC dev, and I like promoting best practices, so yeah I'll put it back.